src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -30,6 +31,7 @@ public CommandResult execute() {
         try {
             final ReadOnlyPerson target = getTargetPerson();
             addressBook.removePerson(target);
+            addressBook.addTagging(new Tagging(target, false));
```

Comment:
Now that your try block is getting longer, it would be good to catch the exceptions on for the specific statements throwing it.